test/system/400-unprivileged-access.bats

```diff
@@ -75,7 +75,7 @@ EOF
     is "$output" "/var/lib/containers/storage" "GraphRoot in expected place"
     GRAPH_ROOT="$output"
     run_podman info --format '{{.Store.RunRoot}}'
-    is "$output" "/var/run/containers/storage" "RunRoot in expected place"
+    is "$output" "/run/containers/storage" "RunRoot in expected place"
```

Comment:
I think you want to stet this. From what I'm seeing, `RunRoot` goes a lot deeper and will require a lot more changes.

Reply:
I changed the match to match .*/run/containers/storage